stack.yaml

```diff
@@ -64,13 +64,3 @@ extra-package-dbs: []
 # extra-lib-dirs: [/path/to/dir]
 #
 # Allow a newer minor version of GHC than the snapshot specifies
-# compiler-check: newer-minor
-
-extra-include-dirs:
-- /usr/local/opt/openssl/include
-extra-lib-dirs:
-- /usr/local/opt/openssl/lib
-
-flags: 
-  HsOpenSSL: 
-    macports-openssl : true
```

Comment:
I would generally recommend to file issues (or even PRs) about the lack of MacOS building docs instead of solving your local problems silently. ;) I'm pretty sure someone else will run into that later and we don't even have any docs to help!

(it's just a general recommendation, no actions required here.)

Reply:
yep, pushed it by mistake, already removed